README.md

```diff
@@ -122,6 +122,9 @@ Other Camunda Cloud & Zeebe related applications
 - [Helm Charts](https://helm.camunda.io/) - [Helm](https://helm.sh/) charts to deploy Zeebe to Kubernetes.
 - [Portainer Templates](https://camunda-community-hub.github.io/zeebe-portainer-templates/) - [Portainer](https://www.portainer.io/) templates to deploy Zeebe to Docker.
 - [zdb](https://github.com/Zelldon/zdb) - Zeebe debug and inspection tool, allows to inspect the log and internal state of Zeebe.
+- [zeebe-worker-java-template](https://github.com/camunda-community-hub/zeebe-worker-java-template) - Minimal template for a [Zeebe](https://github.com/camunda-cloud/zeebe)
```

Comment:
Could also move this to the JobWorker section :shrug:

Reply:
I also thought about it. In the end, the workers section has ready-made workers not templates. So that's why it is here. I hope int he future we have more templates and then can give it its own section.